lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -791,6 +791,49 @@ struct RvalueExprVisitor {
     return visitAssignmentPattern(expr, *count);
   }
 
+  Value visit(const slang::ast::StreamingConcatenationExpression &expr) {
+    SmallVector<Value> operands;
+    for (auto stream: expr.streams()) {
+      auto value = context.convertRvalueExpression(*stream.operand);
+      if (!value)
+        continue;
```

Comment:
```suggestion
      if (!value)
        return {};
```

Reply:
Could you change this to `return {}`, such that errors properly propagate and abort the conversion?